tests/e2e-playwright/Makefile

```diff
@@ -106,6 +106,29 @@ test-sleepers-dev: _check_venv_active ## runs sleepers test on local deploy
 		--autoregister \
 		$(CURDIR)/tests/sleepers/test_sleepers.py
 
+.PHONY: test-platform
+test-platform: _check_venv_active
+	@pytest \
+		-sxvv \
+		--color=yes \
+		--pdb \
+		--product-url=http://$(get_my_ip):9081 \
+		--autoregister \
+		--tracing=on \
+		$(CURDIR)/tests/platform_CI_tests/test_platform.py
+
+.PHONY: test-platform-dev
+test-platform-dev: _check_venv_active  ## runs platform test on local deploy (with PWDEBUG=1)
+	@PWDEBUG=1 pytest \
+		-sxvv \
+		--color=yes \
+		--pdb \
+		--product-url=http://$(get_my_ip):9081 \
+		--headed \
+		--autoregister \
+		--tracing=on \
+		$(CURDIR)/tests/platform_CI_tests/test_platform.py
```

Comment:
alternatively (or maybe additionally) you could add a config for this in `launch.template.json` so you can use the vscode debugger